Dockerfile

```
@@ -1,22 +1,38 @@
-FROM debian:latest
+ARG DEBIAN_VERSION=12
+
+FROM debian:${DEBIAN_VERSION}
+
+ENV DEBIAN_VERSION=12
+
+ENV DEBIAN_FRONTEND=noninteractive
+
 SHELL ["/bin/bash", "-c"]
-RUN source /etc/os-release && apt-get update && apt-get install -y curl wget \
-&& wget -q https://packages.microsoft.com/config/debian/$VERSION_ID/packages-microsoft-prod.deb \
+
+RUN apt-get update -y && apt-get install -y curl wget \
```

If we want to switch to a different release of debian (lite, etc) having the DEBIAN TAG be separate from the version would be ideal. This will always get us the correct version of Debian. 

```suggestion
RUN source /etc/os-release && apt-get update -y && apt-get install -y curl wget \
```